straxen/scada.py

```diff
@@ -63,7 +66,7 @@ def get_scada_values(self,
                          interpolation=False,
                          filling_kwargs={},
```

Comment:
I think this is actually dangerous:
https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Did not see it before

Reply:
Interesting, I was not aware of this behavior. Thanks a lot.